### ISSUE TITLE
Can you provide a contribution guide?

### ISSUE THREAD

**Issue by Fenn-CS** [forwardemail/free-email-forwarding#230]
Thanks for the work you have done so far! Really amazing. Lot's of folk might want to help but starting would be hard with a contribution guide. Also "Feature Requests" could easily be implemented via contributions. My suggestions is for you to :
- Make a contribution guide
- Contribution guide should have architecture of application and how things connect
- Create issues for small things you plan to implement and give them tags, that marks difficulty and also states requirements.
- Such a section should have a todo, like the SMTP server rollout and if possible a roadmap.

I am willing to help but would need to to layout the foundation.

**Comment by shaunwarman** [comment]
Hey @Fenn-CS thanks for this! Forward email is built from https://github.com/ladjs/lad and you can [get started here](https://github.com/ladjs/lad). Agree there should be a contributors guide!

**Comment by niftylettuce** [comment]
@Fenn-CS Yes I will be providing and documenting a proper contribution guide, getting started locally, deploying, etc.  It's on the roadmap for sure.

**Comment by niftylettuce** [comment]
@Fenn-CS Are there any missing features you're looking for that I may not already be aware of?

**Comment by Fenn-CS** [comment]
@niftylettuce You already mentioned what I am looking forward to in your roadmap! A complete `SMTP` server that allows forwarding and we can have mailed by "custom domain", with [Postal-Like](https://github.com/postalhq/postal) features. Unlike Postal `forwardemail` would be much cheaper to host and maintain given the choice of nodejs. :)

**Comment by Fenn-CS** [comment]
@niftylettuce for the record : https://youtu.be/QPWe7yXKJRs
